lib/routes.js

```diff
@@ -25,6 +25,7 @@ const argv = require('minimist')(process.argv.slice(2))
 
 const CLOCK_SKEW = 60 * 1000
 const REQUEST_TTL = 3 * 60 * 1000
+const SERIALIZATION_FAILURE_CODE = '40001'
```

Comment:
This is the second time I've added this const, should I create a file with BD errors consts or should we wait to see if they happen more in the code base?

Reply:
Yeah, if you've got it in two separate files it's a good idea to break it out to a common file. Maybe put that in a new PR.